GraphNeuralNetworks/src/layers/temporalconv.jl

```diff
@@ -824,11 +824,13 @@ end
 
 Flux.@layer :noexpand TGCNCell
 
-function TGCNCell((in, out)::Pair{Int, Int}; kws...)
+function TGCNCell((in, out)::Pair{Int, Int}; 
+                  gate_activation = sigmoid,
+                  kws...)
     conv_z = GNNChain(GCNConv(in => out, relu; kws...), GCNConv(out => out; kws...))
```

Comment:
```suggestion
function TGCNCell((in, out)::Pair{Int, Int}, act = relu;
                  kws...)
    conv_z = GNNChain(GCNConv(in => out, act; kws...), GCNConv(out => out; kws...))
```
The activation that we should allow to customize should be the GCNConv's one. One hardly needs to change the gating activation.